src/PantherTestCaseTrait.php

```diff
@@ -147,8 +147,14 @@ public static function isWebServerStarted(): bool
      *
      * @param array $options see {@see $defaultOptions}
      */
-    protected static function createPantherClient(array $options = [], array $kernelOptions = [], array $managerOptions = []): PantherClient
+    protected static function createPantherClient(array $options = [], array $kernelOptions = [], array $managerOptions = []/*, array $browserArguments = []*/): PantherClient
     {
+        if (\func_num_args() < 4 && __CLASS__ !== (new \ReflectionClass(static::class))->getName() && __CLASS__ !== (new \ReflectionMethod(static::class, __FUNCTION__))->getDeclaringClass()->getName()) {
+            @trigger_error(sprintf('The "%s()" method will have a new "array $browserArguments = []" argument in version 1.1.0, not defining it is deprecated since Symfony 1.0.1.', __METHOD__), \E_USER_DEPRECATED);
```

Comment:
I'm totally making up the version numbers in this message. Somebody should make them real. :)

Reply:
It's always the next MAJOR version where the BC break would occur. So this should be 2.0 instead of 1.1